backend/WP/Gutenberg/blocks/Button.php

```diff
@@ -17,13 +17,33 @@ class Button extends AbstractBlock {
 	public function register() : array {
 		return $this->do_registration(
 			[
-				'description'   => 'Adds a button to your page.',
-				'icon'          => 'admin-links',
-				'category'      => 'common-blocks',
-				'template_type' => 'atom',
-				'template_name' => 'buttons/button',
-				'wrapper_elem'  => 'span',
+				'description'       => 'Adds a button to your page.',
+				'icon'              => 'admin-links',
+				'category'          => 'common-blocks',
+				'template_type'     => 'atom',
+				'template_name'     => 'buttons/button',
+				'wrapper_elem'      => 'div',
+				'class'             => [ 'default-block' ],
+				'alignment_options' => true,
 			]
 		);
 	}
+
+	/**
+	 * Re-format the fields to the format required by the UI component.
+	 *
+	 * @param array   $block The block settings and attributes.
+	 * @param string  $content The block inner HTML (empty).
+	 * @param boolean $is_preview $is_preview True during AJAX preview.
+	 * @param integer $post_id Id of the current post.
+	 * @return array
+	 */
+	protected function get_fields( array $block, string $content, bool $is_preview, int $post_id ) : array {
+		return array_merge(
+			[
+				'button_style' => get_field( 'button_style' ) ?? 'default',
```

Comment:
Should we validate that the `get_field` function exists before using it? Just to avoid PHP errors when ACF is not enabled.

Reply:
The theme has a warning message when ACF is not active https://github.com/wearenolte/lean-theme/blob/master/ThemeSetup.php#L57 but it only appears when activating the theme